ccmbioinfo/lfs
# CARDS-2465: The cards-s3-export module should not be enabled by default

This _Pull Request_ implements CARDS-2465 by removing `cards-s3-export` from the set of CARDS modules that are loaded, by default, at start-up.

To test:

1. Build a self-contained Docker image from this (`CARDS-2465`) branch using the `Utilities/Packaging/Docker/build_self_contained.sh` script. (`./build_self_contained.sh cards/cards:latest`)
2. `cd compose-cluster`
3. `python3 generate_compose_yaml.py --oak_filesystem`
4. `docker-compose build && docker-compose up -d`
5. Visit http://localhost:8080 and login as `admin`:`admin`.
6. Visit http://localhost:8080/system/console/bundles and ensure that the `io.uhndata.cards.s3-export` bundle **is not** present.
7. `docker-compose down && docker-compose rm && docker volume prune -f && ./cleanup.sh`
8. `python3 generate_compose_yaml.py --oak_filesystem`
9. Edit `docker-compose.yml` and add the environment variable `ADDITIONAL_SLING_FEATURES=mvn:io.uhndata.cards/cards-s3-export/0.9.22-SNAPSHOT/slingosgifeature` to the `cardsinitial` container.
10.  `docker-compose build && docker-compose up -d`
11. Visit http://localhost:8080 and login as `admin`:`admin`.
12. Visit http://localhost:8080/system/console/bundles and ensure that the `io.uhndata.cards.s3-export` bundle **is** present.
13. `docker-compose down && docker-compose rm && docker volume prune -f && ./cleanup.sh`
